### PR TITLE
3d City DB code improvement

### PR DESCRIPTION
**Pull request by jailln**
Some comments added and removal of pre-3DCityDB unused code.

### CODE REVIEW

File: building_server/server.py [hunk 1]
```diff
@@ -135,19 +142,25 @@ def run(self, args):
...
+        # Create root node
         root = {
             'box': utils.Box3D(bboxIndex[0]),
             'id': 0,
             'depth': 0,
             'children': []
         }
+
+        ## Construction of the tiles hierarchy (each tile being a node in the
+        ## form of 'root' defined above) by browsing it from the root to the leaves.
```
Comment (Jeremy-Gaillard) [inline building_server/server.py]: One # too many

File: building-server-processdb.py [hunk 2]
```diff
@@ -98,7 +100,13 @@ def initDB(city, conf, scoref):
...
                 continue
 
-            tileId = nextId()
+            # if there is too much features in the current tile, it is subdivided
+            # in subtiles. The tiles with the highest scores go into the parent tile
+            # and the ones with the lowest scores go into the children tiles.
+            # There is no mention of score into the subdivision process because
+            # we use the fact that they are already sorted in descending order
+            # (done in score_for_polygon function of database.py)
+            tileId = nextId() # id of the next tile (to be genereted to have the right number of feature per tile)
```
Comment (Jeremy-Gaillard) [inline building-server-processdb.py]: > to be genereted to have the right number of feature per tile

This is incorrect. The nextId() function generates a unique id for our tiles, it has nothing to do with the number of features per tiles.

File: building_server/server.py [hunk 3]
```diff
@@ -10,7 +10,10 @@
 import json
 import numpy as np
 
+# This file holds the functions of the rest API
 
+# Returns the geometry of a tile
+# args : city ; tile ; format ; attributes (what is that ?)
```
Comment (Jeremy-Gaillard) [inline building_server/server.py]: > (what is that ?)

I think the attribute retrieval is broken at the moment, it has not yet been adapted to 3d citydb.

**Review by Jeremy-Gaillard — CHANGES_REQUESTED** [review]
Thanks for the comments and cleanup. Can you merge the four commits into one? They cancel each-other out, it's confusing.

**Comment by jailln** [comment]
Thanks for the review, I've made the requested changes.

**Pull request merged**